foursight_core/run_result.py

```diff
@@ -224,7 +224,7 @@ def is_not_primary(key):
 
         return num_deleted_s3, num_deleted_es
 
-    def get_result_history(self, start, limit, after_date=None):
+    def get_result_history(self, start, limit, sort, after_date=None) -> [list, int]:
```

Comment:
I don't think this is the right way to type hint a tuple return. Do you have some reference that says it is? My understanding is you have to write: `Tuple[list, int]` using `from typing import Tuple`.

Reply:
Oh, I think you're right, got/took some bad advice somewhere online, will correct these et.al.

Reply:
All set.